File: examples/swagger-3/src/test/resources/asserts/typescript/typescript_feign/src/evt/CreateOrderEvt.ts
/* tslint:disable */
/* eslint-disable */

        import {Sex} from "../enums/Sex";
        import {BaseEvt} from "./BaseEvt";



export interface  CreateOrderEvt extends BaseEvt {

            /**
                *sn 约束条件：输入字符串的最小长度为：0，输入字符串的最大长度为：50
                *字段在java中的类型为：String
            **/
        sn?: string;
            /**
                *字段在java中的类型为：Map
                *字段在java中的类型为：Sex
                *字段在java中的类型为：String
            **/
        test?: Record<'MAN' | 'WOMAN' | 'NONE',string>;
}

File: examples/swagger-3/src/test/resources/asserts/java/spring_cloud_openfeign/src/com/wuxp/codegen/swagger3/evt/CreateOrderEvt.java
package com.wuxp.codegen.swagger3.evt;

import lombok.Data;
import lombok.experimental.Accessors;
import javax.validation.constraints.*;
        import com.wuxp.codegen.swagger3.enums.Sex;
        import java.util.Map;
        import com.wuxp.codegen.swagger3.evt.BaseEvt;

@Data
@Accessors(chain = true)
public class  CreateOrderEvt extends BaseEvt {

            /**
                *sn 约束条件：输入字符串的最小长度为：0，输入字符串的最大长度为：50
                *  ，默认值：，示例输入：
                *字段在java中的类型为：String
            **/
                @Size(
                    max=50
                )
        private String sn;

            /**
                *字段在java中的类型为：Map
                *字段在java中的类型为：Sex
                *字段在java中的类型为：String
            **/
        private Map<Sex,String> test;

}
